src/run_validation.py

```diff
@@ -21,7 +21,7 @@ def main() -> int:
 
     test_result = ValidationTestRunner.dispatch(args, args.distribution).run()
     logging.info(f'final test_result = {test_result}')
-    return test_result  # type: ignore
+    return 0 if test_result else 1  # type: ignore
```

Comment:
What will be the value of test_result? True/False?

Reply:
If it is not then you are just assuming if there is test_result value just return 0 then.

Reply:
Thanks for the review, @peterzhuamazon ..  the test_result will be either True or False which taken from the actual job runner. Becasue the CI was expecting the '0' as Successful instead of 'True'; hence, I wanted to transform 'True' into '0' to satisfy CI.